metals/src/main/scala/scala/meta/internal/metals/SqlSharedIndices.scala

```diff
@@ -0,0 +1,15 @@
+package scala.meta.internal.metals
+
+import java.nio.file.Paths
+
+import scala.meta.io.AbsolutePath
+
+class SqlSharedIndices
+    extends H2ConnectionProvider(
+      () => AbsolutePath(Paths.get(sys.props("user.home"))).resolve(".metals"),
+      "metals-shared",
```

Comment:
Could you used names parameters? Would be useful to see what are these strings used for.